### ISSUE TITLE
[WEB] Static link library

### ISSUE THREAD

**Issue by michaldev** [DavBfr/dart_pdf#694]
Hi. I have problem. When I add libraries like in docs/readme works fine but I have to run my software on web without internet connection then I try like this:

```
  <script src="libs/pdf.min.js"></script>
  <script type="text/javascript">
       pdfjsLib.GlobalWorkerOptions.workerSrc = "libs/pdf.worker.min.js";
  </script>
```

I have files in /build/web and /web/, but When I try to open website I see only white page without any errors in console (js and flutter).

**Comment by michaldev** [comment]
I solved problem. I added js libraries for assets and it fixed this problem.

I have other, but with canvas kit. Not related to this lib.

**Comment by DavBfr** [comment]
@michaldev  Good!
Can you explain a bit more what you did, to help others with the same issue?

**Comment by michaldev** [comment]
Yes.

I added this to pubspec file:
```
 assets:
   - assets/js/pdf.min.js
   - assets/js/pdf.worker.min.js
```

And in HTML:
```
  <script src="assets/assets/js/pdf.min.js"></script>
  <script type="text/javascript">
       pdfjsLib.GlobalWorkerOptions.workerSrc = "assets/assets/js/pdf.worker.min.js";
  </script>
```

This is not PRO option, because mobile package will be include this package. But I didn't find any solutions.

And... It doesn' work, but unrelated to this library. Flutter WEB doesn't work without internet connection because requires canvaskit.

I solved this problem with this code:
assets:
   - assets/js/pdf.min.js
   - assets/js/pdf.worker.min.js
   - assets/canvaskit/
 
And this with build command: 
--dart-define=FLUTTER_WEB_CANVASKIT_URL=assets/assets/canvaskit/

But... It still doesn't work, because canvaskit requires google fonts from internet...
